### ISSUE TITLE
JSON Unescaped bask slashes

### ISSUE THREAD

**Issue by Neo23x0** [hasherezade/pe-sieve#13]
Hi Hasherezade, 

I've tried to process the JSON output in your newest release in LOKI and noticed that back slashes are not escaped. 

![screen shot 2018-04-13 at 08 24 40](https://user-images.githubusercontent.com/2851492/38719891-72e73ac6-3ef4-11e8-8452-260a857177f7.png)

**Comment by hasherezade** [comment]
Indeed, paths are not escaped. I am sorry, I forgot about it. It will be fixed soon.

**Comment by hasherezade** [comment]
Please check the latest builds (attached to the [README](https://github.com/hasherezade/pe-sieve/blob/master/README.md)) and let me know if everything works fine.
